shell.nix

```diff
@@ -22,6 +22,7 @@ pkgs.mkShell {
     bitcoin
     clightning
     jq
+    procps
```

Comment:
Why was this added?

Reply:
`pkill` isn't being identified during docker build so added it as a package in nix-shell